.github/workflows/build_and_test_debug.yml

```diff
@@ -9,6 +9,10 @@ on:
     types:
       - opened
       - synchronize
+  pull_request_target:
```

Comment:
I thought you meant we own't need pull_request_target with this PR

Reply:
ah, this is an outdated - if we put the dev certs in the repo like we discussed, this goes away